CHANGELOG.md

```diff
@@ -126,6 +126,7 @@
 
 ### Jsonnet
 
+* [CHANGE] Removed etcd-operator from the jsonnet. Users can still use etcd as a KV store for rings, but need to deploy and manage etcd themselves rather than via the operator. #13049
```

Comment:
```suggestion
* [CHANGE] Removed etcd-operator from the Jsonnet configuration. Users can still use etcd as a KV store for rings, but need to deploy and manage etcd themselves rather than via the operator. #13049
```